pcli/src/command/query/tx.rs

```diff
@@ -18,7 +18,7 @@ impl Tx {
 
         // generic bounds on HttpClient::new are not well-constructed, so we have to
         // render the URL as a String, then borrow it, then re-parse the borrowed &str
-        let client = HttpClient::new(app.tendermint_url.to_string().as_ref()).unwrap();
+        let client = HttpClient::new(app.pd_url.to_string().as_ref()).unwrap();
```

Comment:
Hmm, this is using the `tendermint_rpc` client to talk to the `pd_url`, don't we want to be using the new grpc proxy instead?

Reply:
good catch